status.tmpl

```diff
@@ -162,67 +222,108 @@
           }
         }
       };
+
+      var countNodeStatuses = function() {
+        var dbs = {{ marshal . }};
+        var elements = document.getElementsByClassName('versionstate');
+        [].forEach.call(elements, function (elem) {
```

Comment:
This is an interesting way to iterate an array. Is `elements` only array-like?

Reply:
Yeah it's an HtmlCollection, so this is the best alternative I could find to `Array.prototype.forEach.call...`